api/controllers/ApplicationController.js

```diff
@@ -355,7 +359,7 @@ var applicationController = {
                     return null;
                   }).catch(function (error) {
                   sails.log(error);
-                  console.log(error);
+                  sails.log.info(error);
```

Comment:
just one of these is needed `sails.log.error(error)`. I suggested using `trace` because you were using the stack trace to debug, but there's no need to print it twice if you're not using it to debug anymore (if that makes sense?)